api/database/models.py

```diff
@@ -12,7 +12,7 @@ class Item(db.Model):
     # Auto-incrementing, unique primary key
     id = Column(Integer, primary_key=True)
     # unique item
-    name = Column(String(80), unique=True, nullable=False)
```

Comment:
We removed this for the duplicate data, but once we re-classify data we can always add this back.